test/wrapper_array.jl

```diff
@@ -72,6 +72,37 @@ end
         @test size(nda[CartesianIndex(1, 1), newaxis]) == (1, )
         @test nda[CartesianIndex(1, 1), newaxis][1] == nda[1, 1]
     end
+
+    # https://github.com/invenia/NamedDims.jl/issues/70
+    @testset "BitArray" begin
```

Comment:
should we replicate this test set, for `Array{Bool}` which should act the same?
```
julia> x = rand(3,2)
3×2 Array{Float64,2}:
 0.241709  0.691836
 0.671663  0.258613
 0.705006  0.366028

julia> x[x .> 0.5]
3-element Array{Float64,1}:
 0.6716626129087626
 0.7050058537208825
 0.6918362123847104

julia> collect(x .> 0.5)
3×2 Array{Bool,2}:
 0  1
 1  0
 1  0


julia> x[collect(x .> 0.5)]
3-element Array{Float64,1}:
 0.6716626129087626
 0.7050058537208825
 0.6918362123847104

```

Reply:
Good idea, it didn't work before, but now does & is tested.